src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,62 @@
+package duke.task;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Represents a deadline item that inherits from Task.
+ */
+public class Deadline extends Task {
+
+    /** Date of when the deadline is due. */
+    protected Date date;
+    /** Boolean that represents if the date contains a time. */
+    protected boolean isTime;
+
+    /**
+     * Creates a deadline with a default not done.
+     * @param description Description of deadline.
```

Comment:
Should there be a line break before describing the parameters? Noticed that some methods have a line break while others do not have.